resources/data/elephpants.json

```diff
@@ -227,15 +227,15 @@
 		{
             "id": 29,
 			"name": "CakePHP",
-			"description": "Red CakePHP",
+			"description": "CakePHP Framework",
 			"sponsor": "CakePHP",
 			"year": 2017,
 			"image": "https://i.imgur.com/CKkne1L.jpg"
 		},
 		{
             "id": 30,
-			"name": "Cake Foundation",
-			"description": "CakeSF",
+			"name": "CakeSF",
```

Comment:
I think Cake Software Foundation is the right name, can you double check please?

Reply:
Cake Software Foundation = CakeSF
and in description all name: Cake Software Foundation.
do you agree?

Reply:
sounds good